src/lib/telldus-accessory.js

```diff
@@ -47,6 +48,12 @@ class TelldusAccessory {
     this.log = string => log(`[${this.name}]: ${string}`)
     this.Characteristic = homebridge.hap.Characteristic
     this.Service        = homebridge.hap.Service
+
+    // Support age limit for sensors to avoid stale information.
+    this.isStale = sensor => {
+      const maxAge = this.data.maxAge || this.config.maxAge || 60 * 60;
```

Comment:
I see, you want `maxAge` to be configurable through the config. I did not draw the parallell the other PR but now I see. Please convert `60 * 60` to a constant at the top of the file for documentation reasons, and I'm definitely fine with the rest.